api/_mixins/SVGFilterPrimitiveStandardAttributes__SVGFEBlendElement.json

```diff
@@ -0,0 +1,246 @@
+{
+  "api": {
+    "SVGFEBlendElement": {
+      "height": {
+        "__compat": {
+          "spec_url": "https://drafts.fxtf.org/filter-effects/#dom-svgfilterprimitivestandardattributes-height",
+          "support": {
+            "chrome": {
+              "version_added": "5"
+            },
+            "chrome_android": {
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "3"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "≤12.1"
```

Comment:
This should fail the lint but doesn't because of https://github.com/mdn/browser-compat-data/issues/9044.

Reply:
The lint errors I can predict should be averted by https://github.com/mdn/browser-compat-data/pull/10648.

Reply:
Thanks, walking the tree would be really cool for mixins (and actually for making sure **all** data trees are consistent).